src/libs/ajax.js

```diff
@@ -298,6 +298,10 @@ const Billing = signal => ({
   listProjectsExtended: async () => {
     const res = await fetchSam('api/resources/v1/billing-project', _.merge(authOpts(), { signal }))
     return res.json()
+  },
+  listUsersOnBillingProject: async projectId => {
```

Comment:
We should match the api structure we use elsewhere, so I call `Billing.project(id).list()` or something